src/odemis/gui/cont/fastem_acq.py

```diff
@@ -76,8 +80,29 @@ def __init__(self, tab_data, tab_panel):
         self.bmp_acq_status_info = self._tab_panel.bmp_acq_status_info
         self.selection_panel = self._tab_panel.selection_panel
 
+        # For dwell time slider
+        self.add_dwell_time_slider = add_dwell_time_slider
+        if self.add_dwell_time_slider:
+            self.current_dwell_time = self._main_data_model.ebeam.dwellTime.value
```

Comment:
`current_dwell_time` doesn't always contain the current value of the dwell time. So it's a confusing name. I'd suggest to call it `orig_dwell_time`. Also add a comment why this is needed. Something like "to restore live stream  dwell time after the overview completed".

That said, saving/restoring this value probably could be avoided by simply making the "dwellTime" VA of the "single beam" stream "local": use `emtvas` instead of `hwemtvas`.